Platform/ElkhartlakeBoardPkg/CfgData/CfgData_Power.yaml

```diff
@@ -368,7 +368,7 @@
       help         : >
                      Enable or Disable PROCHOT# signal being driven externally; 0- Disable; <b>1- Enable</b>.
       length       : 0x01
-      value        : 0x01
+      value        : 0x00

```

Comment:
So we disable PROCHOT# signal being driven externally ?
Is this the requirement for FuSa ?

Reply:
We need to enable ProcHotOut for Fusa requirement.
The UPD is indicating "Disable"ProcHotOut so we're disable this to enable.

Reply:
Its already verified by PMC team with this upd changes.

Reply:
OK. got it.   But does it has impact on other SKU ?    
Do you think it is better to detect FuSa SKU at runtime and then enable it in SBL ?

Reply:
let me check with PMC team again for this

Reply:
Confirmed with PMC team its only enable for Fusa sku.
Patch submitted as well.